front/components/SlackChannelPatternInput.tsx

```diff
@@ -0,0 +1,72 @@
+import { ioTsResolver } from "@hookform/resolvers/io-ts";
+import * as t from "io-ts";
+import React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+
+import { PokeButton } from "@app/components/poke/shadcn/ui/button";
+import { InputField } from "@app/components/poke/shadcn/ui/form/fields";
+
+interface MultiInputProps {
+  initialValues: string;
+  onValuesChange: (value: string) => void;
+}
+
+export const SlackChannelFormSchema = t.type({
+  inputValue: t.string,
+});
+
+export type SlackChannelFormType = t.TypeOf<typeof SlackChannelFormSchema>;
+
+export function SlackChannelPatternInput({
```

Comment:
We def don't want `poke` components in `front/components` 

Let's just inline the code or have a `poke` specific component.

But really here I think we can just put that code in the file where it's used since it won't be reused anywhere else

Reply:
Moving it `front/components/poke` for the sake of readability